### DISCUSSION TITLE
Upgrading SGW

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/DP2SceneGraphWizard#116] [Q&A]
Jumping back in after an absence! I converted to SGW way back, but you've added so many cool tweaks and features! Is there "Best Method" for upgrading my SGW-converted channel? I'm concerned that I might screw something up if I start poking around!

**Comment by rrirower** [comment]
You should tread lightly if you don't understand the BrightScript language. However, if you haven't made any code changes and don't plan to, you can simply re-run the wizard to generate an updated channel with the new features.
